system_tests/bigquery.py

```diff
@@ -602,3 +613,18 @@ def test_dump_table_w_public_data(self):
         dataset = Config.CLIENT.dataset(DATASET_NAME, project=PUBLIC)
         table = dataset.table(TABLE_NAME)
         self._fetch_single_page(table)
+
+    def test_create_table_nested_schema(self):
+        TABLE_NAME = 'test_table'
```

Comment:
> 1.  Not a constant

Huh?  It is as constant as any other name bound to an immutable value.

> 2.    Worth making more unique via unique_resource_id?

The dataset which will contain (only) this table already gets a per-run ID.